pldp/survey/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
If we decide to keep the various separate apps, then be sure to clean up the unneeded files, like this one.